packages/godwoken/index.d.ts

```diff
@@ -0,0 +1,181 @@
+import { HexNumber, Hash, Transaction, Script } from "@ckb-lumos/base";
+export interface SyncParam {
+  reverts: L1Action[];
+  updates: L1Action[];
+  next_block_context: NextBlockContext;
+}
+
+export interface L1Action {
+  transaction_info: TransactionInfo;
+  header_info: HeaderInfo;
+  context: SubmitTxs | StartChallenge | CancelChallenge | Revert;
+}
+
+export interface NextBlockContext {
+  aggregator_id: HexNumber;
+  timestamp: HexNumber;
+}
+
+export interface TransactionInfo {
+  transaction: Transaction;
+  block_hash: Hash;
+}
+
+export interface HeaderInfo {
```

Comment:
Same as the above, it would be better if the JS API only requires Header, instead of HeaderInfo